v22.2/node-shutdown.md

```diff
@@ -264,7 +317,7 @@ If the `cockroach` process has not terminated at the end of the grace period, a
 
 - When using [`systemd`](https://www.freedesktop.org/wiki/Software/systemd/) to run CockroachDB as a service, set the termination grace period with [`TimeoutStopSec`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#TimeoutStopSec=) setting in the service file.
 
-- When using [Kubernetes](kubernetes-overview.html) to orchestrate CockroachDB, set the termination grace period with `terminationGracePeriodSeconds` in the [StatefulSet manifest](deploy-cockroachdb-with-kubernetes.html?filters=manual#configure-the-cluster).
+- When using [Kubernetes](kubernetes-overview.html) to orchestrate CockroachDB, set the termination grace period with `terminationGracePeriodSeconds` in the [StatefulSet manifest](deploy-cockroachdb-with-kubernetes.html?filters=manual#configure-the-cluster). Refer to [Decommissioning and draining on Kubernetes](#decommissioning-and-draining-on-kubernetes).
```

Comment:
As mentioned above, maybe just remove this to avoid duplication with the other section you added?

Reply:
Done